src/main/java/org/nuxeo/ecm/csv/CSVImporterDocumentFactory.java

```diff
@@ -36,6 +36,16 @@ public void createDocument(CoreSession session, String parentPath, String name,
 
     public void updateDocument(CoreSession session, DocumentRef docRef, Map<String, Serializable> values);
 
+    /**
+     * @return {@code true} if a document with the specified parentPath and name exists. {@code false} otherwise.
+     * @since 8.1
+     */
+    public boolean exists(CoreSession session, String parentPath, String name);
+
+    /**
+     * @deprecated since 8.1
+     */
+    @Deprecated
```

Comment:
Currently, the CoreSession does not provide a reliable means by which these values could be used to accomplish the operation. However, I presume the importer itself could use these values locally to accomplish some de-duping of the CSV rows, for example. I will modify my patch per your suggestion.